### PR TITLE
Change UI to be more similar to current extension: App Header

### PR DESCRIPTION
**Pull request by buberdds**
Main requirements were:
- use account icon generator from extension for Account Dropdown
- remove search bar (for now I removed it only from mobile, but I am adding screenshots of two versions with and without search for large screens
- css changes

Screenshot 2022-07-19 at 08 30 50

with search
Screenshot 2022-07-19 at 11 06 13

without search
Screenshot 2022-07-19 at 11 07 13

with dynamic account icon
Screenshot 2022-07-19 at 11 07 58
Screenshot 2022-07-19 at 11 07 51

Screenshot 2022-07-19 at 11 57 10
Screenshot 2022-07-19 at 11 56 45

### CODE REVIEW

File: src/app/components/JazzIcon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export const JazzIcon = memo(({ diameter, seed }: JazzIconProps) => {
+  const ref = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (ref?.current) {
+      const icon = jazzicon(diameter, seed)
+
+      ref.current.replaceChildren()
+      ref.current.appendChild(icon)
```
Comment (lukaw3d) [inline src/app/components/JazzIcon/index.tsx:18]: Is this preferred over `ref.current.replaceChildren(icon)`?

File: src/app/components/JazzIcon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }
+  }, [diameter, ref, seed])
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/components/JazzIcon/index.tsx:20]: What other option I have to inject DOM elements generated by 3rd party, not react lib? I think it's Ok that's what metamask is doing for example.

File: src/types/jazzicon.d.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+declare module '@metamask/jazzicon' {
+  const jazzicon: (number, number) => HTMLDivElement
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/types/jazzicon.d.ts:2]: oh maaaan 😂

File: src/app/components/JazzIcon/index.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/components/JazzIcon/index.tsx]: nah that's my oversight

**Review by tjanez — APPROVED** [review]
The new icon avatar/switcher looks great!
I would remove the search box completely.

File: src/app/components/JazzIcon/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/app/components/JazzIcon/index.tsx]: Yep yep. I can't find docs for the recommended way :shrug:

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**